index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>mentor</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+	<script src="https://kit.fontawesome.com/c9e7c3e582.js" crossorigin="anonymous"></script>		
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<link rel="stylesheet" type="text/css" href="responsivo.css">
+</head>
+<body>
+	<header>
```

Comment:
Acho que aqui dá pra gente fazer o seguinte:
```suggestion
	<header class="header">
```

E remover o `<div class="header">` que tem dentro